xml/snmp_sysdescr.xml

```diff
@@ -2322,6 +2322,21 @@ Copyright (c) 1995-2005 by Cisco Systems
     <param pos="1" name="os.version"/>
   </fingerprint>
 
+  <!--======================================================================
+                            Fortinet FortiGate
+   =======================================================================-->
+
+  <fingerprint pattern="^[\w\-]+\s+v([\d\.]+),build\d+,\d+(?:\s+\(\w+\))?$">
+    <description>FortiGate</description>
```

Comment:
Minor: this could be more descriptive, for example, "Fortinet FortiGate" or "Fortinet FortiGate Network Firewall".